src/OptiHPLCHandler/empower_handler.py

```diff
@@ -179,7 +179,7 @@ def GetEmpowerProjects(self) -> list:
         the user has access to and returns them as a list.
         """
         project_list = self.connection.get("/authentication/project-list")[0]
-        return [item["projectName"] for item in project_list]
+        return project_list
```

Comment:
If you want to be nice, you could change the type of the return in line 175 from `list` to `list[dict[str,str]]`. That will make it a bit easier to use.

Reply:
corrected to List and Dict using the existing typing module 

def GetEmpowerProjects(self) -> List[Dict[str,str]]: